package.json

```diff
@@ -108,19 +99,8 @@
     "logo": "https://opencollective.com/opencollective/logo.txt"
   },
   "config": {
-    "validate-commit-msg": {
```

Comment:
Why has it been removed? scope is optional in our case

Reply:
I have replace deprecated `validate-commit-msg` by `@commitlint/cli` and `@commitlint/config-conventional`.

Reply:
Make sure that commits like: `chore: foo bar` yields at all. We have scope as an optional parameter

Reply:
we need `scope require false` option.

Reply:
Yes

Reply:
It's already optional since we're using `@commitlint/config-conventional`.